src/lib/templateHelpers.ts

```diff
@@ -131,3 +132,14 @@ export const isCommonQueryParameter = (property: string): boolean =>
   property
     ? commonParameterPositions.queryParameters.includes(property.toString())
     : false;
+
+/**
+ * Checks whether a trait name is allowed to be used in the API.
+ * Currently only disallows "offset-paginated", as "OffsetPaginated" is also
+ * available, and is preferred.
+ *
+ * @param trait - Trait to check
+ * @returns true unless the trait's name is "offset-paginated"
+ */
+export const isAllowedTrait = (trait: amf.model.domain.Trait): boolean =>
```

Comment:
Can we add a detailed note that RAMLs are planned to update to use standards and this will remain mainly for backward compatibility

Reply:
done